### PR TITLE
Add github action for kubectl

### PR DESCRIPTION
**Pull request by c-bata**
Hi! Thanks for creating this useful action.

In this PR, I added `kubectl` and `kubectl-config` github actions to operate gke kubernetes clusters.

### CODE REVIEW

File: kubectl/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM gcr.io/cloud-builders/kubectl@sha256:abeaf7bd496e66301f92fd884683feaec6894bfb12d1a02fdbc98a920fab4968
```
Comment (c-bata) [inline kubectl/Dockerfile]: https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/kubectl

**Comment by c-bata** [comment]
I confirmed this github action works properly in following settings.

**Comment by jgcmarins** [comment]
What is missing for this PR to be merged?

File: kubectl-config/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  uses = "actions/gcloud/auth@master"
+  secrets = ["GCLOUD_AUTH"]
+}
+
+action "Set kubectl config" {
+  needs = ["GCP Authenticate"]
+  uses = "actions/gcloud/kubectl-config@master"
+  env = {
+    GCLOUD_ZONE = "your-gcp-region"
+    GCLOUD_PROJCT = "your-gcp-project"
```
Comment (wearethefoos) [inline kubectl-config/README.md]: I think this is a typo as in `kubectl-entrypoint.sh` it says:

```
gcloud config set project "$GCLOUD_PROJECT"
```

So I guess this should be:

```
GCLOUD_PROJECT = "your-gcp-project"
```

Comment (c-bata) [reply kubectl-config/README.md]: Thank you for your review. 

I'll fix this later! The suggested change of Github review is also welcime.

File: kubectl-config/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e
+
+gcloud config set project "$GCLOUD_PROJECT"
+gcloud config set container/use_client_certificate True
```
Comment (wearethefoos) [inline kubectl-config/entrypoint.sh]: Not sure why this is here, but when I tested this on a newly created cluster I needed to remove this line or the script fails consistently with:

```
ERROR: (gcloud.container.clusters.get-credentials) get-credentials requires edit permission on CLUSTER_NAME
```

**Comment by chrispat** [comment]
This actions has been deprecated in favor of https://github.com/GoogleCloudPlatform/github-actions.  This repo has been archived and will be made private on 12/31/2019

**Pull request closed without merging**